src/pybind/mgr/dashboard/frontend/src/app/ceph/block/nvmeof-gateway-group/nvmeof-gateway-group.component.ts

```diff
@@ -74,6 +86,22 @@ export class NvmeofGatewayGroupComponent implements OnInit {
       }
     ];
 
+    const editAction: CdTableAction = {
+      permission: 'read',
+      icon: Icons.edit,
+      disable: () => this.selection.hasMultiSelection,
+      click: () => {},
+      name: this.actionLabels.VIEW_DETAILS
+    };
```

Comment:
is this PR dealing with the editing of the group? if so that needs to be given in the title and description.